quickstart/deploy-mayastor.md

```diff
@@ -6,317 +6,99 @@
+NAMESPACE: mayastor
+STATUS: deployed
+REVISION: 1
+NOTES:
+OpenEBS Mayastor has been installed. Check its status by running:
+$ kubectl get pods -n mayastor
 
-{% tab title="Example Output" %}
-```text
-NAME       DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                                         AGE
-mayastor   3         3         3       3            3           kubernetes.io/arch=amd64,openebs.io/engine=mayastor   108s
+For more information or to view the documentation, visit our website at https://openebs.io.
 ```
 {% endtab %}
 {% endtabs %}
 
-The number and status of mayastor pods can be observed by using the [Mayastor kubectl plugin](https://mayastor.gitbook.io/introduction/reference/kubectl-plugin). Check that the expected number of nodes are reporting their State as `online`
+Verify the status of the pods by running the command:
 
 {% tabs %}
 {% tab title="Command" %}
 ```text
-kubectl mayastor get nodes
-```
-{% endtab %}
-
-{% tab title="Example Output" %}
-```text
-NAME                       STATE    AGE
-aks-agentpool-12194210-0   online   8m18s
-aks-agentpool-12194210-1   online   8m19s
-aks-agentpool-12194210-2   online   8m15s
-```
-{% endtab %}
-{% endtabs %}
-
+kubectl get pods -n mayastor
+```
+{% endtab %}
+{% tab title="Sample Output" %}
+```text
+NAME                                         READY   STATUS             RESTARTS   AGE
+mayastor-agent-core-6d9d498df-kz6t4          1/1     Running            0          25m
+mayastor-api-rest-5c79485686-sw2ll           1/1     Running            0          25m
+mayastor-csi-controller-65d6bc946-qvl2h      3/3     Running            0          25m
+mayastor-csi-node-g8ngz                      2/2     Running            0          25m
+mayastor-csi-node-w86kw                      2/2     Running            0          25m
+mayastor-csi-node-z6lc9                      2/2     Running            0          25m
+mayastor-etcd-0                              1/1     Running            0          25m
+mayastor-etcd-1                              1/1     Running            0          25m
+mayastor-etcd-2                              1/1     Running            0          25m
+mayastor-io-engine-72zp6                     2/2     Running            0          25m
+mayastor-io-engine-gk7z2                     2/2     Running            0          25m
+mayastor-io-engine-wfld7                     2/2     Running            0          25m
+mayastor-loki-0                              1/1     Running            0          25m
+mayastor-obs-callhome-5f47c6d78b-h7x98       1/1     Running            0          25m
+mayastor-operator-diskpool-b64b9b7bb-m7l6c   1/1     Running            0          25m
+mayastor-promtail-cw8lc                      1/1     Running            0          25m
+mayastor-promtail-kbp4t                      1/1     Running            0          25m
+mayastor-promtail-xnhcm                      1/1     Running            0          25m
```

Comment:
Nitpick: The "age" field in the sample output is 25m. Usually, we would want to check the status of pods sooner in the case of installation. You may want to consider reducing the age to 2m.

Reply:
Updated.